templates/communities/communities.js

```diff
@@ -293,26 +317,23 @@ export default async function decorate(doc) {
       twoCols,
     ),
     aside(
-      div('right').innerHTML = rightAside,
+      rightAside,
     ),
   ));
 
-  // place the left right columns after the carousel
-  doc.querySelector('.carousel-wrapper').insertAdjacentElement('afterend', leftRight);
-  // place the breadcrumbs after the carousel
-  doc.querySelector('.carousel-wrapper').insertAdjacentElement('afterend', breadCrumbsEl);
+  mainSection.appendChild(breadCrumbsEl);
+  mainSection.appendChild(leftRight);
+  mainSection.appendChild(modelFilter);
+  mainSection.appendChild(plansAnchor);
+  mainSection.appendChild(filterSectionTitle);
+  mainSection.appendChild(inventoryEl);
```

Comment:
Minor:  Consider using append - I think it reads a little better

mainSection.append(
  breadCrumbsEl, 
  leftRight, 
  modelFilter, 
  plansAnchor, 
  filterSectionTitle, 
  inventoryEl
);

Reply:
fixed.